### PR TITLE
Add a Cancel Button when editing a bill

### LINKED ISSUE #1001: Add `cancel` button when editing a bill
When choosing to edit an existing bill, the only way to cancel the action is to go back in your browser, or to select `submit` without adding any changes. 

ihatemoney-edit-bill-issue

### PR DESCRIPTION
**Pull request by erwan-lacoudre**
Should close #1001

### CODE REVIEW

**Comment by almet** [comment]
Looks good to me 👍

**Comment by erwan-lacoudre** [comment]
> I have just seen that the pull request has not pass the tests that seem related to the database. However I have not touch it, maybe there is something I'm missing ?

@almet do you have any idea ? I don't really see what's wrong

**Comment by almet** [comment]
Hey, it's not related to the work you've done, we have an issue currently on this project which makes the tests not pass. We're currently solving it in #1012. We will merge your changes just after :-)

Thanks, anyways, it seems to be great :-)

File: ihatemoney/templates/forms.html [hunk 1]
```diff
@@ -200,7 +200,11 @@
     </fieldset>
     <div class="actions">
         {{ form.submit(class="btn btn-primary") }}
-        {% if not edit %} {{ form.submit2(class="btn btn-light") }}{% endif %}
+        {% if edit %} 
+            <a href="{{ url_for(".list_bills") }}" class="btn btn-primary"> {{_("Cancel") }} </a>
```
Comment (Glandos) [inline ihatemoney/templates/forms.html:204]:
```suggestion
            <a href="{{ url_for(".list_bills") }}" class="btn btn-outline-secondary"> {{_("Cancel") }} </a>
```

For a better visual UX:
image

**Review by Glandos — CHANGES_REQUESTED** [review]

**Review by Glandos — APPROVED** [review]

**Comment by Glandos** [comment]
Thanks!

**Pull request merged**